src/internal/utils/bucketHelper.ts

```diff
@@ -850,8 +850,9 @@ export const transformMeasureBuckets = (
 ) => {
     let unusedMeasures: IBucketItem[] = [];
 
-    const newBuckets: IBucket[] = measureBucketItemsLimits.map(({ localIdentifier, itemsLimit }) => {
-        const preferredBucketItems = getPreferredBucketItems(buckets, [localIdentifier], [METRIC]);
+    const newBuckets: IBucket[] = measureBucketItemsLimits.map(({ localIdentifiers, itemsLimit }) => {
+        const localIdentifier = localIdentifiers[0] || BucketNames.MEASURES;
```

Comment:
How about `const [localIdentifier = BucketNames.MEASURES] = localIdentifiers`?

Reply:
Updated. Thank you